website-next/tsconfig.json

```diff
@@ -4,24 +4,23 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
+    "strict": false,
```

Comment:
Does this need to be false? I'd like us to keep this as close to the editor's tsconfig as possible

Reply:
fixed